packages/orchestration/src/typeGuards.js

```diff
@@ -18,3 +19,19 @@ export const Proto3Shape = {
 };
 
 export const CoinShape = { value: M.bigint(), denom: M.string() };
+
+/**
+ * - `give` allows any `Nat` `issuerKeyword` record. Must be exactly one entry.
+ * - `exit` must be `{ waived: null }`
+ * - `want` must be empty
+ */
+export const DepositProposalShape = M.splitRecord(
```

Comment:
`waived: null` merits some explanation: once somebody makes such an offer, undoing it may be infeasible, so the client can't honor exit requests; clients must waive their right to exit.

docs should bear a WARNING notice about lack of offer safety

maybe the name should be scarier too... `GiveAndHope` :)

maybe these docs belong on the `deposit()` method and/or the `Deposit` invitationMaker.